src/client/components/app/Transactions/DepositTx.tsx

```diff
@@ -87,9 +87,7 @@ export const DepositTx: FC<DepositTxProps> = ({
       dispatch(
         TokensActions.setSelectedTokenAddress({
           tokenAddress:
-            !zapperEnabled && selectedVault.zapInWith === 'zapperZapIn'
-              ? selectedVault.token.address
-              : selectedVault.defaultDisplayToken,
+            !zapperEnabled && selectedVault.zapInWith ? selectedVault.token.address : selectedVault.defaultDisplayToken,
```

Comment:
We could do here the same refactor as the sdk and rename `zapper`->`zap`, thoughts?

Reply:
yeah agree, seems like best thing to do for now